pkgs/tools/filesystems/unionfs-fuse/default.nix

```diff
@@ -39,7 +47,7 @@ stdenv.mkDerivation rec {
     description = "FUSE UnionFS implementation";
     homepage = "https://github.com/rpodgorny/unionfs-fuse";
     license = licenses.bsd3;
-    platforms = platforms.linux;
+    platforms = platforms.linux ++ platforms.darwin;
```

Comment:
```suggestion
    platforms = platforms.unix;
```

Reply:
If I understand it right that would enable all unixes while that pr only fixes darwin build. https://github.com/NixOS/nixpkgs/blob/4d100429b57ea7406576c736448e9921ac4bdffc/lib/systems/inspect.nix#L38  Do we have some agreement that platform.unix is basically linux + darwin?

Reply:
> Do we have some agreement that platform.unix is basically linux + darwin?

Basically it is. platforms is mostly for hydra and we do not have any other OS on hydra and BSD, SunOS, Cygwin and Redox is mostly untested all over nixpkgs.